_includes/head.html

```diff
@@ -3,7 +3,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   {% if page.layout == "eip" %}
-    <title>EIP {{ page.eip }}: {{ page.title | escape }}</title>
+    <title>EIP-{{ page.eip }}: {{ page.title | escape }}</title>
     <meta property="og:title" content="EIP {{ page.eip }}: {{ page.title | escape }}" />
```

Comment:
```suggestion
    <meta property="og:title" content="EIP-{{ page.eip }}: {{ page.title | escape }}" />
```

Reply:
And more instances of this to be found in the metadata below.

Reply:
Should be update now, thanks for pointing those out.